Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -39,7 +39,7 @@ final class APIDiffTests: XCTestCase {
       }
     }
 
-    func testSimpleAPIDiff() throws {
+    func _DISABLE_testSimpleAPIDiff() throws {
```

Comment:
Seems better to use one of the variants of XCTSkip(), as is done in some other tests that are failing.

Reply:
+1

Reply:
done